docs/intro.rst

```diff
@@ -1,8 +1,165 @@
 Introduction
 ============
 
-.. toctree::
+CumulusCI helps development teams build great applications on the Salesforce platform by automating org setup, testing, and deployment.
 
-   learn_more
-   faq
-   read_next
+
+
+Automation with CumulusCI
+-------------------------
+
+If your product development lifecycle and release process is anything like ours at Salesforce.org, it's complex. You're managing multiple packages, dependencies, orgs, and release versions. Not to mention managing org metadata and all the setup operations that need to run in the right sequence, before or after a package is installed, to create a properly configured org. 
+
+Consider the example of Nonprofit Success Pack (NPSP), one of Salesforce.org's flagship open source products. NPSP is a large, complex, heterogeneous application. It consists of six managed packages, with complex dependency relationships. Using automation, all five dependent packages are deployed in the right sequence, the unpackaged record types for the Account and Opportunity objects are delivered, and the final configurations to make the customer's experience better, such as setting up Global Actions and delivering translations, are performed. Biweekly NPSP releases are released to tens of thousands of customer orgs -- with the proper configurations, and without requiring end users to work through a lengthy setup guide.
```

Comment:
"Biweekly NPSP releases are released..."

Would deployed be better just so we're not using releases so close together with released?

Reply:
Ya it does sound a little weird but I'm not sure "deployed" is fitting in this context.